sympy/utilities/tests/test_wester.py

```diff
@@ -2181,21 +2182,24 @@ def test_U4():
     assert diff(x**n, x, n).rewrite(factorial) == factorial(n)
 
 
-@XFAIL
-def test_U5():
-    # https://github.com/sympy/sympy/issues/6681
-    # f(g(x)).diff(x,2) returns Derivative(g(x), x)**2*Subs(Derivative(
-    #  f(_xi_1), _xi_1, _xi_1), (_xi_1,), (g(x),)) + Derivative(g(x), x, x)*
-    #  Subs(Derivative(f(_xi_1), _xi_1), (_xi_1,), (g(x),))
-    raise NotImplementedError("f(g(t)).diff(t,2) Subs not performed")
+def test_U5():  # XXX check
+    # issue 6681
+    t = symbols('t')
+    a1 = (
+        Derivative(f(g(t)), g(t))*Derivative(g(t), (t, 2)) +
+        Derivative(f(g(t)), (g(t), 2))*Derivative(g(t), t)**2)
+    a2 = (
+        Derivative(f(g(t)), g(t))*Derivative(g(t), (t, 2)) +
+        Derivative(g(t), t)*Derivative(f(g(t)), g(t), t))
+    assert f(g(t)).diff(t,2) in (a1, a2)
 
 
-@XFAIL
-def test_U6():
+def test_U6():  # XXX check
     h = Function('h')
-    # raises ValueError: Invalid limits given: (y, h(x), g(x))
     T = integrate(f(y), y, h(x), g(x))
-    T.diff(x)
+    assert T.diff(x) == (
+        h(x)*Derivative(g(x), x)*Integral(f(y), y) +
+        g(x)*Derivative(h(x), x)*Integral(f(y), y))
 
```

Comment:
The derivative should be `f(g(x)) * Derivative(g(x), x) - f(h(x)) * Derivative(h(x), x)`